### ISSUE TITLE
Map doesnt work in older versions of IE

### ISSUE THREAD

**Issue by devninjajeremy** [NewSignature/us-map#29]
The map doesnt work at all in IE8 or 7. Even the main page doesn't work correctly. Is anyone else having this issue?

**Comment by webgoon** [comment]
Yeah, most mobile phones it doesnt work neither.   The page is empty when
viewing may be related to the IE 7or 8 issue.

**Comment by jdsteinbach** [comment]
The map is drawn as an svg image - IE 8 (and down) doesn't support svg images, so no map for them.

**Comment by kcaran** [comment]
I haven't had any issues with IE8 or IE7. The raphael.js plugin converts SVG to VML, which the older browsers can understand.

**Comment by eweisbrot** [comment]
I'm having the same issue, as the map doesn't display when using IE8. Any ideas how to fix?

**Comment by JamesSingleton** [comment]
I am having issues with the map showing on IE11 Version 11.0.9600.18499
